components/views/dropdown-menu-view.js

```diff
@@ -0,0 +1,261 @@
+define([
+        'backbone',
+        'jquery',
+        'underscore',
+        '../utils/constants',
+        'text!../templates/dropdown.underscore'
+    ],
+    function( Backbone, $, _, constants, DropdownTpl ) {
```

Comment:
Nit: I'm thinking about what the variable should be called. In edx-platform we would name this ``Constants`` or ``ToolkitConstants`` or something. I'm fine with going with it this way too for now, but I figured I'd at least raise the thought.